Cargo.toml

```diff
@@ -35,10 +36,13 @@ wasmtime-wast = { path = "lib/wast" }
 docopt = "1.0.1"
 serde = "1.0.75"
 serde_derive = "1.0.75"
-faerie = "0.6.0"
+faerie = "0.7.0"
 target-lexicon = { version = "0.2.0", default-features = false }
 pretty_env_logger = "0.3.0"
 file-per-thread-logger = "0.1.1"
 wabt = "0.7"
 
 [workspace]
+
+[patch.crates-io]
+gimli = { git = "https://github.com/gimli-rs/gimli.git", rev = "955c33ed" }
```

Comment:
Currently using newest version of the gimli: it allows writing functionality.

/cc @fitzgen, @philipc